quartz/components/ContentMeta.tsx

```diff
@@ -48,7 +55,13 @@ export default ((opts?: Partial<ContentMetaOptions>) => {
   .content-meta {
     margin-top: 0;
     color: var(--gray);
+
+    > span[show-comma="true"]:not(:last-child){
```

Comment:
I'm ok with this, but ik @jackyzha0 might have a saying on this.

Also we should probably move content-meta to a scss sheet separately.

Reply:
yup let's use a scss stylesheet, this is just pure CSS and only some browsers support nested selectors like this (scss will do the syntax downleveling for us)

Reply:
@jackyzha0  should I move this back in to base.scss then?